edc-extensions/migrations/control-plane-migration/src/main/resources/org/eclipse/tractusx/edc/postgresql/migration/dataplaneinstance/V0_0_1__Init_Dataplaneinstance.sql

```diff
@@ -0,0 +1,34 @@
+--
+--  Copyright (c) 2024 Contributors to the Eclipse Foundation
+--
+--  This program and the accompanying materials are made available under the
+--  terms of the Apache License, Version 2.0 which is available at
+--  https://www.apache.org/licenses/LICENSE-2.0
+--
+--  SPDX-License-Identifier: Apache-2.0
+--
+--  Contributors:
+--       Contributors to the Eclipse Foundation - initial API and implementation
+--
+
+
+CREATE TABLE IF NOT EXISTS edc_lease
+(
+    leased_by      VARCHAR NOT NULL,
+    leased_at      BIGINT,
+    lease_duration INTEGER NOT NULL,
+    lease_id       VARCHAR NOT NULL
+        CONSTRAINT lease_pk
+            PRIMARY KEY
+);
+
+
+CREATE TABLE IF NOT EXISTS edc_data_plane_instance
+(
+    id                   VARCHAR NOT NULL PRIMARY KEY,
+    data                 JSON,
+    lease_id             VARCHAR
+        CONSTRAINT data_plane_instance_lease_id_fk
+                    REFERENCES edc_lease
+                    ON DELETE SET NULL
+);
```

Comment:
Do the schemas of these tables match the old tables, so that in case of an upgrade of the edc (even if it is not supported) there should be no problems?